source/locales/pages/en-us/diffuser.md

```diff
@@ -1,68 +1,59 @@
-# Disseminate Our Climate Actions
+# Disseminate Nos Gestes Climat
 
-Nos Gestes Climat is a public service, its code [is
-free](/documentation) and reusable by everyone.
+Nos Gestes Climat is a public service, its code <a href="/documentation">is free</a> and reusable by everyone.
 
-However, you can take advantage of and distribute our simulator in
-several ways without mobilizing development skills.
+However, you can take advantage of and distribute our simulator in several ways without mobilizing development skills.
 
-## You want to integrate the simulator directly on your website
+## You want to share a page in a specific language
 
-Nothing could be easier! Integrate it into a blog post or site simply by
-adding the line below to your web page.
+Since the version <a href="https://nosgestesclimat.fr/nouveautés/glacier-du-géant">Glacier du Géant</a>, the interface of the Nos Gestes Climat website is translated into several languages. You can therefore share a page of the site in a specific language by indicating at the end of the the URL `?lang=en`.
 
-`<script id="nosgestesclimat" src="https://nosgestesclimat.fr/iframe.js"></script>`
+For example, to share the page _/diffuser_ in English :
 
-Join [the many current
-reusers](https://datagir.ademe.fr/apps/nos-gestes-climat/) in this way.
+`https://nosgestesclimat.fr/diffuser?lang=en`
 
-The module will automatically stay up to date with the [latest
-developments](/nouveautés).
+The currently supported languages are French (`fr`) and English (`en`).
 
-To learn more about integration, visit [our
-guide](https://github.com/datagir/nosgestesclimat-site/blob/master/PERSONNALISATION.md).
+> Note that when you change the interface language, the NGC's model remains unchanged. It is only the text of the site that is translated and not the NGC's model itself.
```

Comment:
I suggest to use "calculation model" instead of "NGC's model" and change the last sentence into : "Only the website text is translated"

Reply:
Reworded :heavy_check_mark: